economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+
+\begin{equation}
+\label{phaseswitch}
+    T_{1\rightarrow2}= \frac{S_{Ph1}}{S_0 \cdot \kappa^* \cdot I_{max}}
+\end{equation}
+\\
+Where $\kappa^*$ is the average lock duration of all sub-stakes, weighted by relative size, and $S_{Ph1}$ is the total number of tokens reserved for Phase 1, excluding the genesis supply (roughly 1.83 billion). For example, if $\kappa^*$ were to equal 11 months, $T_{1\rightarrow2}$ would be  5 years and 79 days. The longest Phase 1 can last (i.e. if all sub-stakes unlock in 1 month or less, so $\kappa^* \approx 0.54$), barring changes to the protocol, is 8 years and 209 days.
+
+\begin{figure}[h!]
+    \includegraphics[width=\textwidth]{graphs/SubsidyP1.pdf}
+    \caption{Cumulative total subsidies received in \textbf{Phase 1}, for stakes unlocking in 12+ and 1 month, and configured to re-stake or withdraw subsidies. Staking rate \& average lock duration assumed to be 70\% and 0.77 respectively.}
+    \label{fig:tp}
+\end{figure}
+
+\begin{figure}[h!]
+    \includegraphics[width=\textwidth]{graphs/SubsidyP2.pdf}
+    \caption{Cumulative total subsidies received in approx. first 7 years of \textbf{Phase 2}, with all other parameters the same as figure 4. Note that all totals at the end of Phase 1 equal those at the start of Phase 2.}
+    \label{fig:tp}
+\end{figure}
+
+\subsection{Other considerations for service-providers}
+
+\subsubsection{Re-staking}
+
+A service-provider is free to withdraw the subsidies their sub-stakes receive. If they pocket all incoming tokens, then their sub-stakes will remain the same size, but are likely to decrease as a fraction of the total tokens staked ($\Theta$), which means their period-to-period subsidy, and exposure to user requests, will also decrease. 
+\\\\
+A service-provider may re-stake their subsidies, provided they have the capacity to support the potential increase in workload (user requests). If the sub-stakes in question are set to unlock in 1 year or longer, and the average lock duration is shorter than this, then re-staking all subsidies would see those sub-stakes grow and account for a increasing fraction of the total tokens staked. As a result, the subsidies received would also rise over time. \\\\
+The dramatic difference in long-term  cumulative earnings between various staking strategies over both phases is shown in figures 4 and 5. 
+
+\subsubsection{Winding down sub-stakes}
+
+The default setting for all sub-stakes is for $t_i$, the stake unlock duration, to be automatically prolonged. This keeps the subsidy coefficient ($\kappa$) constant. If a service-provider proactively initiates a wind-down – that is, they specify an unlock date in the future for a sub-stake, then the corresponding subsidy will continually decrease as the time to unlock gets smaller and smaller, until $\kappa$ reaches a minimum of $0.5$.
```

Comment:
This keeps the subsidy coefficient... -> This keeps the staking duration constant and subsequently the subsidy coefficient...

```suggestion
The default setting for all sub-stakes is for $t_i$, the stake unlock duration, to be automatically prolonged. This keeps the staking duration constant and subsequently the subsidy coefficient ($\kappa$) constant. If a service-provider proactively initiates a wind-down – that is, they specify an unlock date in the future for a sub-stake, then the corresponding subsidy will continually decrease as the time to unlock gets smaller and smaller, until $\kappa$ reaches a minimum of $0.5$.
```

Reply:
Just realised my mistake – `t_i` is not the duration but the moment in time the sub-stake unlocks. Prolonging applies to a duration not a date, so removing it. 
```suggestion
The default setting for a sub-stake $i$ is for $D_i$, the duration of time until the unlock date, to be automatically prolonged. This holds the subsidy coefficient ($\kappa_i$) constant. If a service-provider proactively initiates a wind-down – that is, they specify an unchanging unlock date in the future, then the corresponding subsidy will continually decrease as the duration to unlock gets smaller and smaller, until $\kappa_i$ reaches a minimum of $0.5$.
```